apps/notification/mutation.py

```diff
@@ -0,0 +1,48 @@
+from django.utils.translation import gettext
+
+import graphene
+
+from utils.graphene.mutation import generate_input_type_for_serializer
+from utils.graphene.error_types import mutation_is_not_valid, CustomErrorType
+
+from .serializers import NotificationGqSerializer
+from .schema import NotificationType
+from .models import Notification
+
+NotificationStatusInputType = generate_input_type_for_serializer(
+    'NotificationStatusInputType',
+    serializer_class=NotificationGqSerializer
+
+)
+
+
+class NotificationStatusUpdate(graphene.Mutation):
+    class Arguments:
+        data = NotificationStatusInputType(required=True)
+
+    errors = graphene.List(graphene.NonNull(CustomErrorType))
+    ok = graphene.Boolean()
+    result = graphene.Field(NotificationType)
+
+    @staticmethod
+    def mutate(root, info, data):
+        try:
+            instance = Notification.objects.get(id=data['id'])
```

Comment:
Let's filter notification queries by users or add validation in serializer or both.